tests/ThentosSpec.hs

```diff
@@ -69,13 +69,17 @@ spec = do
 
     describe "UpdateUser" $ do
       it "changes user if it exists" $ \ (st, _, _) -> do
-        result <- update' st $ UpdateUser (UserId 1) user1 allowEverything
+        result <- update' st $ UpdateUserField (UserId 1)
+                                               (UpdateUserFieldName "fka_user1")
+                                               allowEverything
         result `shouldBe` Right ()
         result2 <- query' st $ LookupUser (UserId 1) allowEverything
-        result2 `shouldBe` (Right (UserId 1, user1))
+        result2 `shouldBe` (Right (UserId 1, user1 {_userName = "fka_user1"}))
```

Comment:
you can always go to the import list in `Thentos.Trans.DB` and then search the module for how the operators mentioned there are used.  that should get you far enough.  (-: